.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-- 0.1
 - 4.3
 - stable
```

Comment:
This is probably a good time to update this to something similar to Chai core (although even that is a bit outdated since it doesn't explicitly list 8 and 9; currently picked up by safety nets):

```
 - 4 # to be removed 2018-04-01
 - 6 # to be removed 2019-04-01
 - 7 # to be removed 2017-06-30
 - lts/* # safety net; don't remove
 - node # safety net; don't remove
```

Reply:
done